hardhat.config.ts

```diff
@@ -62,6 +61,33 @@ const DEFAULT_CONTRACT_COMPILER_SETTINGS = {
   },
 };
 
+const networks = Object.fromEntries(
+  Object.values(CHAIN_IDs)
+    .filter((chainId) => PUBLIC_NETWORKS[chainId] !== undefined)
+    .map((chainId) => {
+      const network = PUBLIC_NETWORKS[chainId];
+      const name = network.name.replace(" ", "").toLowerCase();
```

Comment:
nb. might need to add some special handling for the `-sepolia` chains.

Reply:
In https://github.com/across-protocol/contracts/pull/833/commits/61f2d29f02457163c926b10ee0ddbd97a65a99f8 I reverted to using the chainId as the network key, rather than a symbolic name. The problem with symbolic names is that we have no consistent naming scheme for the various networks. Chain IDs are at least universally consistent and don't impose decisions like whether to hyphenate spaces or not.